core/src/main/java/org/frankframework/validation/xsd/StringXsd.java

```diff
@@ -21,6 +21,8 @@
 import java.io.StringWriter;
 
 import lombok.Setter;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Deze wil je niet gebruiken lijkt me

Reply:
Nee dat willen we niet, is er ook alweer uit.
Maar IntelliJ zette hem er zelf in bij refactorings en deze was erdoor geglipt.

Zie ook mijn post in Teams. 😃